mods/translucent-windows.wh.cpp

```diff
@@ -0,0 +1,494 @@
+// ==WindhawkMod==
+// @id              translucent-windows
+// @name            Translucent Windows
+// @description     Enables native translucent effects on windows
+// @version         1.0
+// @author          Undisputed00x
+// @github          https://github.com/Undisputed00x
+// @include         *
+// @compilerOptions -ldwmapi -luxtheme
+// ==/WindhawkMod==
+
+// ==WindhawkModReadme==
+/*
+
+### Acrylic (BlurBehind)
+![BlurBehind](https://i.imgur.com/S8OGRvc.png)
+### Acrylic (SystemBackdrop)
+![Acrylic SystemBackdrop](https://i.imgur.com/66cAiEC.png)
+### Mica
+![Mica](https://i.imgur.com/7BGDhJa.png)
+### MicaAlt
+![MicaTabbed](https://i.imgur.com/EeviFqO.png)
+
+# FAQ
+
+* Prerequisited windows settings to enable the background effects
+    - Transparency effects enabled
+    - Energy saver disabled
+#
+* Extending effects to the entire window can result in text being unreadable or even invisible in some cases. 
+Light mode theme, HDR enabled or white background behind the window can cause this. 
+This is because some GDI rendering operations do not preserve alpha channel values.
+
+* Acrylic blur behind effect may show a bleeding effect at the edges of a window when 
+maximized or snapped to the edge of the screen, this is caused by default by the windows DWMAPI.
+
+*/
+// ==/WindhawkModReadme==
+
+// ==WindhawkModSettings==
+/*
+- ThemeBackground: TRUE
+  $name: Optimize windows theme
+  $description: Fill with black color the file explorer background in order to render a clear translucent effect
+- type: none
+  $name: Effects
+  $description: Windows 11 version >= 22621.xxx (22H2) is required
+  $options:
+  - none: Default
+  - acrylicblur: Acrylic (BlurBehind)
+  - acrylicsystem: Acrylic (SystemBackdrop)
+  - mica: Mica
+  - mica_tabbed: MicaAlt
+- ExtendFrame: TRUE
+  $name: Extend effects into entire window

```

Comment:
Here's how the mod affects the stock Windows theme with the default settings:

![image](https://github.com/user-attachments/assets/1a106344-a2cc-456b-9546-99d10a974516)

Perhaps the default settings can be tweaked for the common setup. For example, maybe it's better to have the `ExtendFrame` option disabled by default. What do you think?

Especially given that now disabling the mod doesn't revert the changes, it can result in a bad experience of having to reopen all windows to make them usable again.

Reply:
These settings worked for me with a stock theme:

![image](https://github.com/user-attachments/assets/0c32c86a-0cd9-4b79-882c-14e07a4ceb31)